devel/juno-1/start.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+fh_bin="firehose-cosmos"
+
+if [[ -z $(which $fh_bin || true) ]]; then
+  echo "You must install the firehose-binary first. See README for instructions"
```

Comment:
just a small piece. May make more sense for this to say "firehose-cosmos binary" rather than "firehose-binary"